scopes/dependencies/dependency-resolver/dependency-resolver.docs.md

```diff
@@ -1,235 +1,251 @@
 ---
-description: Dependency management for Bit components.
-labels: ['dependency', 'dependencies', 'component', 'bit']
+description: Simplifies the dependency management of components in a Bit workspace
+labels: ['core aspect', 'dependencies']
 ---
 
-This extension is responsible for:
-1. detecting dependencies of components by static code analysis
-2. apply dependencies polices
-3. resolve dependencies versions
-4. calculate the final dependencies of a given component
-
-## Usage
-
-### workspace config
-The extension's workspace configuration will have the following fields:
-* policy
-* packageManager
-* strictPeerDependencies
-* packageManagerArgs
-Here is a full example of workspace config:
-(For complete reference see DependencyResolverWorkspaceConfig type)
-```js
-/**
-  * main configuration for component dependency resolution.
-  */
+The Dependency Resolver salvages us from the tedious and time-consuming task of defining the dependency graph for every single component in a Bit workspace.
+It does so by automatically generating their dependency graph and by offering an efficient API to manually modify the generated graph using policies that can be applied on groups of components.
+
+#### Features
```

Comment:
add a section about the fact that bit install will use bit registry by default (unless your default in npmrc are not npmjs) 
and also that the token is taken from the @bit in the npmrc (soon from the bit global config as well)